ansible/doc/README.testbed.VsSetup.md

```diff
@@ -16,6 +16,7 @@ $ sudo brctl addbr br1
 $ sudo ifconfig br1 10.250.0.1/24
 $ sudo ifconfig br1 up
 ```
+- Install Ansible
```

Comment:
You don't need install Ansible in the host, and there is already Ansible inside docker-sonic-mgmt. Could you double check?

Reply:
Good point. There should be no need to manually install Ansible.

Reply:
You're right, I double checked- I unnecessarily installed Ansible on the host after receiving `ansible-playbook command not found` error due to some issues with my local environment